lib/src/components/NxDonutChart/NxDonutChart.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { FunctionComponent, ReactElement } from 'react';
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+const NxDonutChart: FunctionComponent<Props> =
+  function NxDonutChart(props): ReactElement<Props> {
+    const decimal = props.percent / 100;
+    const diameter = props.outerRadius * 2;
+    const strokeWidth = props.outerRadius - props.innerRadius;
+    const r = props.innerRadius + strokeWidth / 2;
+    const circumference = 2 * Math.PI * r;
+    const strokeDasharray = [decimal * circumference, (1.0 - decimal) * circumference];
+    const strokeDashoffset = (0.25 * circumference) - (1.0 - decimal) * circumference;
+    return (
+      <svg width={diameter} height={diameter}>
+        <circle cx={props.outerRadius}
+                cy={props.outerRadius}
+                strokeWidth={strokeWidth}
+                r={r}
+                fill="transparent"
```

Comment:
Can be in css.  Also "none" would probably be the more appropriate value.

Reply:
> Can be in css. Also "none" would probably be the more appropriate value.

Moved in https://github.com/sonatype/sonatype-react-shared-components/pull/172/commits/5a3e4fd020df42fb5582f6d3dd79160c5e40120b